go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/observatorium/opa-openshift
 
-go 1.18
+go 1.19
```

Comment:
Does this project actually require golang 1.19 or depend on any libraries that require that? If not, then this doesn't need to be bumped. I'm fact, maybe it doesn't even need to be 1.18. what version of go is required and what version of go this project is built with are two different things.

Reply:
I agree that it makes sense for libraries to depend on the "lowest version possible", but for binaries I tend to keep it in sync with the version that it is currently built against.